internal/util/rand.go

```diff
@@ -2,21 +2,34 @@ package util
 
 import (
 	"math/rand"
+	"time"
 )
 
-func UniqRands(l int, n int) []int {
-	set := make(map[int]struct{})
-	nums := make([]int, 0, l)
-	for {
-		num := rand.Intn(n)
-		if _, ok := set[num]; !ok {
-			set[num] = struct{}{}
-			nums = append(nums, num)
-		}
-		if len(nums) == l {
-			goto exit
-		}
+func init() {
+	rand.Seed(time.Now().UnixNano())
```

Comment:
The rand seed is set in `apps/nsqd/nsqd.go`, so we should not do that here.

I think we don't need to set a "good" random seed for tests (and they will be somewhat more deterministic if we don't).